qemu/tests/openflow_acl_test.py

```diff
@@ -73,21 +73,18 @@ def access_service(access_sys, access_targets, disabled, host_ip, ref=False):
                     out = ""
                     out_err = ""
                     try:
-                        out = remote_login(
+                        stat, out = remote_login(
```

Comment:
Now the function is not only for remote login, telnet will try to login the target, socat only check if port is reachable, how about rename it to adapt your change? `is_port_open`, `verify_port_reachable` or you have a better idea.